WebContent/getAllStudents.jsp

```diff
@@ -1,70 +0,0 @@
-<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
-	pageEncoding="ISO-8859-1"%>
-<%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
-<%@ page import="com.sundar.studentmanagement.student.vo.*"%>

```

Comment:
Why do you need this ??
Hereafter, don't import any java classes here
Use only JSP tags